agent/Gopkg.toml

```diff
@@ -0,0 +1,50 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  name = "github.com/aws/aws-sdk-go"
+  version = "1.8.19"
```

Comment:
What do you think about locking everything to the commit that is referenced in the old Godeps.json file? It would potentially reduce the blast radius.

Reply:
what about locking dependencies to tags and upgrade to nearest tagged version where it makes sense. ill add a separate commit for each dependency i lock to a new version. cause as we were saying earlier, tagging to revisions doesn't really provide guarantees. we can definitely do some things to reduce blast radius though.

Reply:
Assuming that the Gopkg.lock contains the same commits as the previous Gopkg.json file had for our deps I think it's okay to migrate this way.